localstack/testing/pytest/fixtures.py

```diff
@@ -1978,3 +1979,54 @@ def factory(**kwargs):
             appsync_client.delete_graphql_api(apiId=api)
         except Exception as e:
             LOG.debug(f"Error cleaning up AppSync API: {api}, {e}")
+
+
+@pytest.fixture
+def assert_host_customisation(monkeypatch):
+    hostname_external = f"external-host-{short_uid()}"
+    localstack_hostname = socket.gethostname()
```

Comment:
I think this deserves a comment to make it understandable why `socket.gethostname` is necessary here.

Reply:
Great suggestion, thanks!